Aptoide/aptoide-client-v8
# [MOB-649] Remove social links

**What does this PR do?**

This PR aims at removing all social links from the Store profiles.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] ManageStoreFragment.java

**How should this be manually tested?**

Open stores, check if everything is ok. Also try to edit store and change that every thing is ok.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-649](https://aptoide.atlassian.net/browse/MOB-649)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
